pkg/gofr/gofr.go

```diff
@@ -335,36 +335,13 @@ func (a *App) getExporter(name, host, port, url string) (sdktrace.SpanExporter,
 		err      error
 	)
 
-	authHeader := a.Config.Get("TRACER_AUTH_KEY")
-
 	switch strings.ToLower(name) {
+	case "otlp":
+		return a.buildOtlp(url, host, port, strings.ToLower(name))
 	case "jaeger":
-		if url == "" {
-			url = fmt.Sprintf("%s:%s", host, port)
-		}
-
-		a.container.Logf("Exporting traces to jaeger at %s", url)
-
-		opts := []otlptracegrpc.Option{otlptracegrpc.WithInsecure(), otlptracegrpc.WithEndpoint(url)}
-
-		if authHeader != "" {
-			opts = append(opts, otlptracegrpc.WithHeaders(map[string]string{"Authorization": authHeader}))
-		}
-
-		exporter, err = otlptracegrpc.New(context.Background(), opts...)
+		return a.buildOtlp(url, host, port, strings.ToLower(name))
```

Comment:
Here you have to add a comment to explain why jaeger uses open telemetry one, because it looks like a bug.

Many have no idea if/how they are related